lib/shoulda/matchers/active_record/association_matcher.rb

```diff
@@ -1394,17 +1394,23 @@ def touch_correct?
         end
 
         def class_has_foreign_key?(klass)
+          @missing = validate_foreing_key(klass)
+
+          @missing.nil?
+        end
+
+        def validate_foreing_key(klass)
```

Comment:
`validate_foreign_key`?

Reply:
I'm not sure about this change. It seems to me that the `?` Should be used in methods that either return true or false, but in this case, this method is returning an error message - when there is a problem.

Would you like the name to be changed?

Reply:
Done!

Reply:
Haha yeah sorry I should have been more clear :)